service/controller/app/v1/resource/status/create.go

```diff
@@ -45,6 +45,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		AppVersion: chartStatus.AppVersion,
 		Release: v1alpha1.AppStatusRelease{
 			LastDeployed: *chartStatus.Release.LastDeployed.DeepCopy(),
```

Comment:
FMI why does that need a deep copy?

Reply:
I think it was @kopiczko who introduce this practice on chart-operator, can you tell me the reason?

Reply:
I don't remember me doing that and if I did I don't remember why :D

Reply:
When there is no reason we should drop it and keep it simple. [`LastDeployed`](https://godoc.org/github.com/giantswarm/apiextensions/pkg/apis/application/v1alpha1#AppStatusRelease) is only `DeepCopyTime`.

Reply:
Do you mind if I open another PR for that, I would like to keep the subject like this

Reply:
Sure